vmp/helper/tf_test.go

```diff
@@ -0,0 +1,88 @@
+package helper_test
+
+import (
+	"terraform-provider-vmp/vmp/helper"
+	"testing"
+)
+
+func TestInterfaceToStringArray(t *testing.T) {
+
+	cases := []struct {
+		input    interface{}
+		expected []string
+	}{
+		{
+			input:    []interface{}{"val1", "val2", "val3"},
```

Comment:
If input is something like this. new(Person).(interface{}), it may throw a runtime error.